docs/index.md

```diff
@@ -5,52 +5,42 @@ page_keywords: documentation, docs,  docker, compose, orchestration, containers
 
 ## Overview
 
-Compose is a tool that allows you to orchestrate multiple Docker containers.
-With Compose, you can build clusters of containers which provide the resources
-(services, volumes, etc.) needed to build and run a complete distributed
-application.
+Compose is a tool for defining and running complex applications with Docker.
+With Compose, you define a multi-container application in a single file, then
+spin your application up in a single command which does everything that needs to
+be done to get it running.
 
-You can use Compose to build your app with containers hosted locally, or on a
-remote server, including cloud-based instances - anywhere a Docker daemon can
-run. Its primary use case is for development environments, but it can be used
-just as easily for staging or CI.
+Compose is especially good for development environments, staging servers and CI.
+It can be used in production, but it isn't (yet) recommended that you do so.
+We're actively working on making it better in that regard - see the
+[roadmap](https://github.com/docker/fig/blob/master/ROADMAP.md).
```

Comment:
I think we need a separate section in the documentation to describe this in more detail, as explained in #944. We shouldn't land users on the roadmap in the second paragraph because that is only going to confuse them (they don't know what Compose really is yet).